diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/CommentsFormatting.kt

```diff
@@ -317,10 +317,11 @@ class CommentsFormatting(configRules: List<RulesConfig>) : DiktatRule(
         }
     }
 
-    private fun isFirstComment(node: ASTNode) = if (node.isBlockOrClassBody()) {
+    private fun isFirstComment(node: ASTNode) = if (node.isChildOfBlockOrClassBody()) {
         // In case when comment is inside of a function or class
         if (node.treePrev.isWhiteSpace()) {
-            node.treePrev.treePrev.elementType == LBRACE
+            // in some cases (e.g. kts files) BLOCK doesn't have a leading brace
+            node.treePrev?.treePrev?.elementType == LBRACE
```

Comment:
`treePrev`, `treeParent` are nullable types, however, they don't marked as such in source code, so IDEA don't advise to make such calls 'save' automatically, and so we use them is some dangerous way
May be we need to change such calls in whole project to appropriate `!!` or `?.` in aim to avoid some problems in future?

Reply:
The problem here is that we won't get any compiler checks, unless source of ASTNode is annotated with [proper annotations](https://kotlinlang.org/docs/java-interop.html#null-safety-and-platform-types). For sure, adding safe access to all such unvokations may save us from some NPEs, but I don't see a good way to enforce this style.